tests/golden.csv

```diff
@@ -753,7 +1138,11 @@ Mike Izbicki,seattletimes.com,2020-05-30,https://www.seattletimes.com/nation-wor
 Mike Izbicki,semana.com,2020-05-31,https://www.semana.com/,FALSE,,,,
 Mike Izbicki,semana.com,2020-05-31,https://www.semana.com/mundo/articulo/,FALSE,,,,
 Mike Izbicki,semana.com,2020-05-31,https://www.semana.com/mundo/articulo/en-grave-peligro-hermetismo-sobre-estado-de-salud-de-kim-jong-un-tras-cirugia/664851,TRUE,es-co,2020-04-20 22:29:00,2020-04-21 23:01:47,¿En grave peligro? Hermetismo sobre estado de salud de Kim Jong-un tras cirugía
-,sfweekly.com,2020-06-06,https://www.sfweekly.com/,FALSE,,,,
```

Comment:
Is there a reason this line is deleted?

Reply:
On my version, that row has just been moved down after inserting the new urls from line 1141 to line 1145.

Reply:
Ahh, I see.  You're right.  Ignore my previous comment :)